plugins/lookup/aws_secret.py

```diff
@@ -117,7 +122,16 @@ def run(self, terms, variables, **kwargs):
         secrets = []
         for term in terms:
             params = {}
-            params['SecretId'] = term
+            if kwargs.get('nested'):
+              # assumes term is of the form `aws_secret_name.key1.key2` where `aws_secret_name` is the secret objects name
```

Comment:
Could you run a **pylint** test before submitting solving the reported suggestions? For example, here, **" Found 14 spaces, expected 16 (bad-indentation)"**

Reply:
Sure, fixed in 1585e87.